ios/RNSScreen.mm

```diff
@@ -2,6 +2,7 @@
 
 #import "RNSScreen.h"
 #import "RNSScreenContainer.h"
+#import "RNSScreenStack.h"
```

Comment:
Why is it needed here?

Reply:
`RNSScreenNavigationController` interface is defined it `RNSScreenStack.h` header and [it is required in `RNSScreenView` implementation](https://github.com/software-mansion/react-native-screens/blob/0c34961e4e87ee0563f11eb17b7fd675c0eb13dd/ios/RNSScreen.mm#L532)